### PR TITLE
Return correct type of aggregation function

### PR DESCRIPTION
**Pull request by sachdevs**
See #32.

### CODE REVIEW

File: src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractAggregator.java [hunk 1]
```diff
@@ -68,10 +68,10 @@ protected void aggregateDouble(int index, double doubleValue) {
     /**
      * @param index index of an aggregate.
      *
-     * @return {@link String} representation of the aggregate value at the given index.
+     * @return {@link Object} long value of the aggregate value at the given index.
      */
-    public String getStringValue(int index) {
-        return "" + longValues[index];
+    public Object getValue(int index) {
```
Comment (nare469) [inline src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractAggregator.java]: Why not return a `long`?

File: src/main/java/ca/waterloo/dsg/graphflow/query/planner/OneTimeMatchQueryPlanner.java [hunk 2]
```diff
@@ -14,7 +14,6 @@
 import ca.waterloo.dsg.graphflow.query.operator.descriptor.EdgeOrVertexPropertyDescriptor;
 import ca.waterloo.dsg.graphflow.query.operator.descriptor.EdgeOrVertexPropertyDescriptor.DescriptorType;
 import ca.waterloo.dsg.graphflow.query.operator.filter.Filter;
-import ca.waterloo.dsg.graphflow.query.operator.filter.LiteralPredicateFactory;
```
Comment (nare469) [inline src/main/java/ca/waterloo/dsg/graphflow/query/planner/OneTimeMatchQueryPlanner.java]: nice

File: src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractAggregator.java (hunk 1 above)
Comment (nare469) [reply src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractAggregator.java]: nvm i get it

File: src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/Average.java [hunk 3]
```diff
@@ -39,11 +39,16 @@ private void adjustSizeAndIncrementCount(int index) {
         this.counts[index] += 1;
     }
 
+    /**
+     * @param index index of an aggregate.
```
Comment (nare469) [inline src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/Average.java]: not sure if this is a typo. can you reword if it isn't?

File: src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractLongDoubleAggregator.java [hunk 4]
```diff
@@ -17,11 +17,17 @@ protected AbstractLongDoubleAggregator() {
         doubleValues = new double[0];
     }
 
+    /**
+     * @param index index of an aggregate.
```
Comment (nare469) [inline src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractLongDoubleAggregator.java]: same as last comment

File: src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/Average.java (hunk 3 above)
Comment (sachdevs) [reply src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/Average.java]: 👍 fixed

File: src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractLongDoubleAggregator.java (hunk 4 above)
Comment (sachdevs) [reply src/main/java/ca/waterloo/dsg/graphflow/query/operator/aggregator/AbstractLongDoubleAggregator.java]: 👍

**Pull request merged**